packages/contracts-bedrock/scripts/outputs/DeleteOutput.s.sol

```diff
@@ -0,0 +1,314 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import { console } from "forge-std/console.sol";
+import { IMulticall3 } from "forge-std/interfaces/IMulticall3.sol";
+
+import { LibSort } from "../libraries/LibSort.sol";
+import { IGnosisSafe, Enum } from "../interfaces/IGnosisSafe.sol";
+import { IL2OutputOracle } from "../interfaces/IL2OutputOracle.sol";
+import { EnhancedScript } from "../universal/EnhancedScript.sol";
+import { GlobalConstants } from "../universal/GlobalConstants.sol";
+
+import { Types } from "../../contracts/libraries/Types.sol";
+import { FeeVault } from "../../contracts/universal/FeeVault.sol";
+import { Predeploys } from "../../contracts/libraries/Predeploys.sol";
+
+/**
+ * @title DeleteOutput
+ * @notice Deletes an output root from the L2OutputOracle.
+ * @notice Example usage is provided in the README documentation.
+ */
+contract DeleteOutput is EnhancedScript, GlobalConstants {
```

Comment:
Hmm this could just inherit `SafeBuilder` right?

Reply:
The idea is that scripts don't need to implement `run`, they just implement building the calldata and post checks

Reply:
Minimal implementation required:

```solidity
// SPDX-License-Identifier: MIT
pragma solidity 0.8.15;

import { SafeBuilder } from "./SafeBuilder.sol";

contract BedrockUpgrade is SafeBuilder {
    function setUp() external {
        //
    }

    function buildCalldata(address _proxyAdmin) internal override view returns (bytes memory) {
        //
    }

    function _postCheck() internal override {
        //
    }

    function test_script_succeeds() skipWhenNotForking external {
        //
    }
}
```

Reply:
Ah yes forgot to refactor to use the `SafeBuilder`, my b. Will do that in this round of fixes b4 re-requesting a review.